aspnetcore/client-side/import-export-interop.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Client-side development with .NET JavaScript `[JSImport]`/`[JSExport]` interop
+author: pavelsavara
+description: Learn how to perform JavaScript interop in client-side JavaScript apps using .NET JavaScript `[JSImport]`/`[JSExport]` interop.
+monikerRange: '>= aspnetcore-7.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 10/11/2022
+uid: client-side/import-export-interop
+---
+# Client-side development with .NET JavaScript `[JSImport]`/`[JSExport]` interop
```

Comment:
Technically you can use .NET from JS in non-client environments, like a Node.js app.

```suggestion
# Run .NET from JavaScript using `[JSImport]`/`[JSExport]` interop
```